MediaPlayer/MediaSlider.module.less

```diff
@@ -0,0 +1,42 @@
+@import "../styles/mixins.less";
+@import "../styles/variables.less";
+@import "../styles/skin.less";
+
+.mediaSlider {
+	cursor: pointer;
+
+	.knob {
+		visibility: hidden;
+	}
+
+	.focus({
+		.knob {
+			visibility: visible;
+		}
+	});
+
+	.applySkins({
+		&.slider {
+			.bar {
+				background-color: @agate-mediaslider-bar-color;
+			}
+
+			.fill {
+				background-image: @agate-mediaslider-fill-bg-image;
+			}
+		}
+
+		.focus({
+			&.slider {
+				.bar {
+					background-color: @agate-mediaslider-bar-color;
```

Comment:
Let's make `@agate-mediaslider-focused-bar-color` and `@agate-mediaslider-focused-fill-bg-image`.